### PR TITLE
Listbook/home

### PR DESCRIPTION
**Pull request by EstebanIndiveri**
## Summary
Add FlatListBook component to HomeScreen
## Screenshots

[Change!] Show the screenshots of the views you modified.

### Android
<a href="https://ibb.co/WFcrCF2">screenshot-1620076306892</a>
Android 9

### iOS
<a href="https://ibb.co/dL9NTyy">listview-Ios</a>
## Trello Card
[Lista de Libros](https://trello.com/c/hq3mte5e/9-n-62-lista-de-libros)

### CODE REVIEW

File: wbooks/src/app/components/FlatListHome/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  id: number;
+}
+
+export default function FlatListHome(): ReactElement {
+  return (
+    <FlatList
+      data={BOOKS_MOCK}
+      renderItem={({ item }: { item: BookInerface }) => (
+        <CardBook author={item.author} image={item.imageUrl} title={item.title} id={item.id} />
+      )}
```
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/FlatListHome/index.tsx]: extract renderItem to const

File: wbooks/src/app/components/FlatListHome/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      showsVerticalScrollIndicator={false}
+      keyExtractor={item => item.id.toString()}
```
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/FlatListHome/index.tsx]: same

File: wbooks/src/app/components/FlatListHome/index.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+import { FlatList } from 'react-native';
+import { BOOKS_MOCK } from '@constants/mockBooks';
+import CardBook from '@components/CardBook';
+
+interface BookInerface {
```
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/FlatListHome/index.tsx]: `BookInterface`

File: wbooks/src/app/components/FlatListHome/index.tsx (hunk 2 above)
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/FlatListHome/index.tsx]: What is type of item?

File: wbooks/src/app/components/FlatListHome/index.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  author: string;
+  imageUrl: string | null;
+  title: string;
+  id: number;
+}
+
+export default function FlatListHome(): ReactElement {
```
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/FlatListHome/index.tsx]: Replace `FlatListHome` by something more especific, Example `BookList`

**Review by andreszuluaga-wolox — CHANGES_REQUESTED** [review]

File: wbooks/src/app/components/BookList/index.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  author: string;
+  imageUrl: string | null;
+  title: string;
+  id: number;
+}
+
+export default function BookList(): ReactElement {
+  const renderBookList: ListRenderItem<BookInterface> = ({ item }) => (
+    <CardBook author={item.author} image={item.imageUrl} title={item.title} id={item.id} />
+  );
```
Comment (mattgle) [inline wbooks/src/app/components/BookList/index.tsx]:
```suggestion
  const renderBookList: ListRenderItem<BookInterface> = ({ item: {title, imageUrl, author, id} }) => (
    <CardBook author={author} image={imageUrl} title={title} id={id} />
  );
```

**Review by mattgle — CHANGES_REQUESTED** [review]

**Review by mattgle — APPROVED** [review]

**Pull request merged**